test/integration/config/test/index.test.js

```diff
@@ -115,10 +127,19 @@ describe('Configuration', () => {
     let browser
     try {
       browser = await webdriver(context.appPort, '/webpack-css')
-      const pTag = await browser.elementByCss('.hello-world')
-      const initialFontSize = await pTag.getComputedCss('font-size')
 
-      expect(initialFontSize).toBe('100px')
+      await check(
+        async () => {
+          const pTag = await browser.elementByCss('.hello-world')
+          const initialFontSize = await pTag.getComputedCss('font-size')
+          return initialFontSize
+        },
+        {
+          test(content) {
+            return content === '100px'
+          },
+        }
```

Comment:
```suggestion
        },
        false
```

We should update this to not throw an un-catchable error everywhere since the clean-up won't be able to be triggered since the error is thrown inside of a timeout and then `jest` fails the test